metadata-ingestion/src/datahub/integrations/great_expectations/action.py

```diff
@@ -686,10 +687,36 @@ def get_dataset_partitions(self, batch_identifier, data_asset):
                         batch_spec_type=type(ge_batch_spec)
                     )
                 )
+        elif isinstance(data_asset, SqlAlchemyDataset):
+            if "." in data_asset._table.name:
+                # bigquery case
+                schema_name, table_name = data_asset._table.name.split(".")
+                sqlalchemy_uri = f"{data_asset.engine.url}/{schema_name}"
```

Comment:
does the engine url already have the project name in it?

Reply:
the engine full url is like `bigquery://my-project`
Then it is parsed to fetch the project name and the database, that's why I've added the `/{schema_name}` here